src/game/Tactical/DisplayCover.cc

```diff
@@ -435,6 +435,45 @@ void DisplayRangeToTarget(SOLDIERTYPE const* const s, INT16 const sTargetGridNo)
 	}
 }
 
+void DisplayChanceToHitTarget(SOLDIERTYPE const* const s, INT16 const sTargetGridNo)
+{
+	if (sTargetGridNo == NOWHERE || sTargetGridNo == 0) return;
+
+
+	SOLDIERTYPE* pSoldier = const_cast<SOLDIERTYPE*>(s);
+
+	// Get the range to the target location
+	UINT16 const usRange = GetRangeInCellCoordsFromGridNoDiff(s->sGridNo, sTargetGridNo) / 10;
+
+	if (WeaponInHand(s))
+	{
+		// Get the chance to hit
+		UINT32 const uiHitChance = CalcChanceToHitGun(pSoldier, sTargetGridNo, pSoldier->bAimTime, pSoldier->bAimShotLocation, true );
```

Comment:
`fModify`, the last arg to `CalcChanceToHitGun`, should be `false` unless you are actually firing the gun. Passing `true` means to modify the status (wearing off) the weapon and attachments.

Reply:
Got it, will change.